world/main.cpp

```diff
@@ -413,37 +465,48 @@ int main(int argc, char** argv) {
 	LogInfo("Deleted [{}] stale player corpses from database", database.DeleteStalePlayerCorpses());
 
 	LogInfo("Loading adventures");
-	if (!adventure_manager.LoadAdventureTemplates())
-	{
+	if (!adventure_manager.LoadAdventureTemplates()) {
 		LogInfo("Unable to load adventure templates");
 	}
 
-	if (!adventure_manager.LoadAdventureEntries())
-	{
+	if (!adventure_manager.LoadAdventureEntries()) {
 		LogInfo("Unable to load adventure templates");
 	}
 
 	adventure_manager.LoadLeaderboardInfo();
 
+	LogInfo("Purging expired dynamic zones and members");
+	dynamic_zone_manager.PurgeExpiredDynamicZones();
+
 	LogInfo("Purging expired expeditions");
 	ExpeditionDatabase::PurgeExpiredExpeditions();
 	ExpeditionDatabase::PurgeExpiredCharacterLockouts();
 
+	LogInfo("Purging expired character task timers");
+	CharacterTaskTimersRepository::DeleteWhere(database, "expire_time <= NOW()");
+
 	LogInfo("Purging expired instances");
 	database.PurgeExpiredInstances();
 
 	Timer PurgeInstanceTimer(450000);
 	PurgeInstanceTimer.Start(450000);
 
-	LogInfo("Loading active expeditions");
-	expedition_state.CacheAllFromDatabase();
+	LogInfo("Loading dynamic zones");
+	dynamic_zone_manager.CacheAllFromDatabase();
 
 	LogInfo("Loading char create info");
 	content_db.LoadCharacterCreateAllocations();
 	content_db.LoadCharacterCreateCombos();
 
+	LogInfo("Initializing [EventScheduler]");
 	event_scheduler.SetDatabase(&database)->LoadScheduledEvents();
 
+	LogInfo("Initializing [SharedTaskManager]");
+	shared_task_manager.SetDatabase(&database)
+		->SetContentDatabase(&content_db)
+		->LoadTaskData()
```

Comment:
Add a purge for expired?

Reply:
Potential follow up here to actually normalize shared task expire time on the shared task object and persistence before we merge

Reply:
Added